_health-care/_js/utils/validations.js

```diff
@@ -190,7 +192,20 @@ function isValidSection(completePath, sectionData) {
   }
 }
 
+function initializeNullValues(value) {
+  if (value === null) {
+    return '';
+  } else if (_.isPlainObject(value)) {
```

Comment:
Nevermind, looks like `_.isPlainObject` won't return `true` for an array, that's good to know :)